### ISSUE TITLE
Failing to create volume 'Opts' unexpected keyword.

### ISSUE THREAD

**Issue by 7imbrook** [ClusterHQ/flocker#2122]
I'm getting the same error referenced in [FLOC-3258](https://clusterhq.atlassian.net/browse/FLOC-3258) with the latest installs of everything.
```
$ flocker-docker-plugin --version
1.5.0

$ docker version                             
Client:                                    
 Version:      1.9.0                       
 API version:  1.21                        
 Go version:   go1.4.2                     
 Git commit:   76d6bc9                     
 Built:        Tue Nov  3 17:54:53 UTC 2015
 OS/Arch:      linux/amd64                 
                                           
Server:                                    
 Version:      1.9.0                       
 API version:  1.21                        
 Go version:   go1.4.2                     
 Git commit:   76d6bc9                     
 Built:        Tue Nov  3 17:54:53 UTC 2015
 OS/Arch:      linux/amd64
```

**Comment by wallnerryan** [comment]
Hi @7imbrook, yes we are aware of this :+1: Our docker plugin does work 1.9, however wasn't included in ```1.5.0```. You should see and update to the ```flocker-docker-plugin``` soon to support this. I will update this issue when it is released and you can verify and we can close. thanks

**Comment by 7imbrook** [comment]
:+1:

**Comment by itamarst** [comment]
Hi, we just released 1.6.1 which should fix this problem - give it a try. Thanks again for the bug report, and let us know if you have more problems, questions or suggestions.

**Comment by 7imbrook** [comment]
Curious as to when this will be released in apt or yum repositories?
